src/store/calendars.js

```diff
@@ -401,7 +401,7 @@ const getters = {
 	},
 
 	hasTrashBin(state) {
-		return state.trashBin !== undefined
+		return state.trashBin !== undefined && state.trashBin._props['{http://nextcloud.com/ns}trash-bin-retention-duration'] !== 0
```

Comment:
`state.trashBin._props['{http://nextcloud.com/ns}trash-bin-retention-duration']` can be undefined as well, in which case we should hide it too.

Reply:
Should we? I thought about this scenario and this can only mean the server doesn't send the prop. We now have https://github.com/nextcloud/calendar/pull/3146 for that.

Reply:
Yeah but things can always go wrong in-between. 

I'm already not so fond of accessing `_props` like this, so instead we could use `_exposeProperty` in cdav and do a simpler check here.

Reply:
> e could use `_exposeProperty` in cdav and do a simpler check here

Let me do that.

Reply:
https://github.com/nextcloud/cdav-library/pull/482